modules/ROOT/pages/mobile.adoc

```diff
@@ -12,16 +12,16 @@ The mobile experience comes with a number of user interface changes in addition
 * Mobile devices now use the `silver` theme.
 * Horizontal contextual menus on mobile.
 * Side-scrolling toolbars  and contextual menus on mobile.
-* Contextual keyboard settings for dialogs using link:{baseurl}/ui-components/dialogcomponents/#inputmode[`inputMode`].
-* <<mobiledefaultsforselectedsettings,Mobile defaults for selected settings>>.
+* Contextual keyboard settings for dialogs using xref:ui-components/dialogcomponents.adoc#inputmode[`inputMode`].
```

Comment:
This xref doesn't resolve as the `inputmode` id doesn't exist. It looks like the id is missing in the `dialogcomponents.adoc` -> `inputMode` heading.

- [ ] Fix broken xref

Reply:
Done! I added the missing id in dialogcomponents.adoc